Update download links and blog post for Pulsar v1.119.0 (#277)
pulsar-edit/pulsar-edit.github.io: docs/blog/20240717-confused-Techie-v1.119.0.md

---
title: "Pulsar v1.119.0 is Live!"
author: confused-Techie
date: 2024-07-17
category:
  - dev
tag:
  - release
---

Pulsar [1.119.0](https://github.com/pulsar-edit/pulsar/releases/tag/v1.119.0) is available now!

<!-- more -->

## Pulsar v1.119.0 is live!

While a smaller release this time around, v1.119.0 still manages to pack a punch.

For macOS, we've gone to great lengths to ensure Pulsar should build just fine on macOS 13+, while our Linux users get greater compatibility for DevTools on various platforms. For our programmers, there's been more of the constant incremental improvements to various languages' built-in syntax highlighting and code folding this time around, with a focus on PHP, Python, Javascript, Typescript, Shell script, and C.

As always thanks a ton to all of those that support the project and keep it moving forward, we appreciate you all, and look forward to seeing you amongst the stars.

\- The Pulsar Team

> ---
> 
> ## **NOTICE**
> 
> Originally, the binaries from our normal "Pull Request" CI build (which produces unsigned binaries) were accidentally uploaded to this release, instead of the binaries from the special "tag push" CI build (which signs the macOS binaries). In order to provide you with signed macOS binaries, we are re-uploading the Intel macOS binaries and updating `SHA256SUMS.txt` to reflect this. As such the following binaries have been swapped out for the correct versions, with the checksum being updated as well for the following files:
> 
> - `Intel.Mac.Pulsar-1.119.0-mac.zip`
> - `Intel.Mac.Pulsar-1.119.0.dmg`
> 
> ---

Reviewer: ```suggestion
---

::: info Notice

Originally, the binaries from our normal "Pull Request" CI build (which produces unsigned binaries) were accidentally uploaded to this release, instead of the binaries from the special "tag push" CI build (which signs the macOS binaries).

In order to provide you with signed Intel macOS binaries, we are re-uploading them and updating `SHA256SUMS.txt` to reflect this.

The following binaries have been swapped out for the correct versions and the checksums updated:

- `Intel.Mac.Pulsar-1.119.0-mac.zip`
- `Intel.Mac.Pulsar-1.119.0.dmg`

:::
```

We should probably use an infobox here to draw attention to the notice. I've also attempted a little rewording and formatting to reduce some repetition and make it a bit easier to read.

Author: I committed it with a rephrase of the present-tense "we are re-uploading" stuff to past tense "we have re-uploaded", and so on. And removed one of the separator `---` lines, since in my view the Notice infobox belongs with the main prose passage, IMO, not the bullet points of the "individual changes" section.
